db/migrate/20180228090121_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :fname
```

Comment:
@jynti Please use valid attributes names:

For example: fname should have been first_name.